docs/src/development/proposals/MEP18/README.md

```diff
@@ -0,0 +1,146 @@
+- Optional: Container Registry to host all metal-stack and gardener containers
+- Optional: Let's Encrypt [boulder](https://github.com/letsencrypt/boulder) as a certificate authority
+- ...
+
+Physical view, minimal setup for a initial cluster with a single physical node:
+
+![Small Initial Cluster](autonomous-control-plane-images/small-initial-cluster.drawio.svg)
+
+Physical View, bigger ha setup which is spread across two data centers:
+
+![HA Initial Cluster](autonomous-control-plane-images/ha-initial-cluster.drawio.svg)
+
+### Control Plane High Availability
+
+Running the initial control plane on a single physical server is not as available as it should be in such a use case. It should be possible to survive a loss of this server, because the server could be lost by many events, such as hardware failure, disk corruption or even failure of the datacenter location where this server is deployed.
+
+Setting up a second server with the same software components is an option, but the problem of data redundancy must be solved, because neither the gardener control plane, nor the metal-stack control plane can be instantiated twice.
+
+Given that we provide part of the local storage of the server as backing storage for the stateful sets in the k3s cluster, the data stored on the server itself must be replicated to another server and backed up on a regular basis.
+
+The replication of ETCD can be achieved through [clustered configuration](https://docs.k3s.io/datastore/ha-embedded) of k3s. Components of metal-stack and Gardener can run standalone and already utilize backup-restore mechanism that must be configured accordingly. For two or more bare metal machine used for the initial cluster, a loadbalancing mechanism for the ingress is required. kube-vip could be a possible solution.
+
+For monitoring a backend like a Victoria Metrics Cluster would allow spearding the monitoring data across the initial cluster nodes. These metrics should also be backed up in object storage.
+
+### Partition
+
+The partition which is managed by the initial cluster can be a simple and small hardware setup but yet capable enough to host the target cluster. It would even be a good practice to create separate target clusters on the initial cluster, e.g. one for the metal-stack control plane and one for the Gardener (maybe one more for monitoring).
+
+It can follow the metal-stack minimal setup which provides about 8-16 small servers connected to a 1G/s or 10G/s network dataplane. Central storage is optional as the persistence of the services running in these clusters is always backed up to a central object storage. Operations would be much easier if a central storage is provided.
+
+## Target Cluster
+
+The target cluster is the metal-stack environment which serves for end-user production use, the control plane is running in a shoot hosted in the initial cluster. The seed(s) and shoot(s) for end-users are created on the machines provided by the target cluster.
```

Comment:
The location of the initial cluster is not clear if we have multiple failure domains (partitions).